src/lib/mina_wire_types/pickles/pickles.mli

```diff
@@ -59,6 +59,62 @@ module Concrete_ : sig
     end
   end
 
+  module Wrap_wire_proof : sig
+    module Columns_vec = Pickles_types.Vector.Vector_15
+    module Coefficients_vec = Pickles_types.Vector.Vector_15
+    module Quotient_polynomial_vec = Pickles_types.Vector.Vector_7
+    module Permuts_minus_1_vec = Pickles_types.Vector.Vector_6
+
+    module Commitments : sig
+      module V1 : sig
+        type t =
+          { w_comm :
+              (Pasta_bindings.Fp.t * Pasta_bindings.Fp.t)
+              Columns_vec.Stable.V1.t
+          ; z_comm : Pasta_bindings.Fp.t * Pasta_bindings.Fp.t
+          ; t_comm :
+              (Pasta_bindings.Fp.t * Pasta_bindings.Fp.t)
+              Quotient_polynomial_vec.Stable.V1.t
```

Comment:
Looks like we might want to give a name (or a least an alias just for the interface) to the type `Pasta_bindings.Fp.t * Pasta_bindings.Fp.t`. Would that be a point?